src/evaluator.ts

```diff
@@ -92,6 +105,72 @@ export class Evaluator {
     }
   }
 
+  private isTargetInList(target: Target, targets: Target[]): boolean {
+    return (
+      targets.find((elem) => elem.identifier === target.identifier) != undefined
```

Comment:
You can simplify with `includes`
```suggestion
      return targets.includes(target)
```

Reply:
Actually, since the vanilla includes is pretty readable, maybe this method isn't required